doc/changes/10650.md

```diff
@@ -0,0 +1,3 @@
+- Re_export dune-section private library in the dune-site library stanza,
+    in order to avoid failure when generating and building sites modules
+    with implicit_transitive_deps = false. (#10650, @MA0100)
```

Comment:
Please remove the extra leading whitespace.

Reply:
Done.